modules/ob-release-notes-1-2.adoc

```diff
@@ -5,20 +5,69 @@
 [id="ob-release-notes-1-2_{context}"]
 = Release notes for {builds-shortname} General Availability 1.2
 
-{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} <version>.
+{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.17.
 
 [id="new-features-1-2_{context}"]
 == New features
 
-The following sections highlight what is new in {builds-shortname} 1.2.
+The following section highlights what is new in {builds-shortname} 1.2.
 
 [id="builds-new-features-1-1_{context}"]
-=== Builds
+=== {builds-shortname}
 
+* With this update, you can use the {builds-title} on the {ocp-product-title} cluster that runs on ARM, IBM Power, and IBM Z worker hardware. The components to build images with Shipwright and Shared Resource CSI Driver are available.
```

Comment:
Indicate this is **Technology Preview**?

```suggestion

* Technology Preview: With this update, you can use the {builds-title} on the {ocp-product-title} cluster that runs on ARM, IBM Power, and IBM Z worker hardware. The components to build images with Shipwright and Shared Resource CSI Driver are available.
```

Reply:
Yes, if it is a tech preview, it should definitely be indicated :+1: